package.json

```diff
@@ -6,67 +6,46 @@
   "license": "GPL",
   "private": true,
   "scripts": {
-    "dev": "WEBPACK_ENV=development webpack --progress --colors --watch --hide-modules",
-    "dev:build": "WEBPACK_ENV=development webpack --progress --colors --hide-modules",
-    "build": "WEBPACK_ENV=production webpack",
-    "makepot": "grunt makepot",
-    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
-    "version": "node bin/version-replace"
+    "start": "wp-scripts start --progress",
+    "start:hot": "wp-scripts start --progress --hot",
+    "build": "wp-scripts build --progress",
+    "packages-update": "wp-scripts packages-update",
+    "version": "node bin/version-replace",
+    "makepot": "node bin/makepot",
+    "readme": "wp-readme-to-md",
+    "clean": "node bin/clean-build-dir",
+    "clean-files": "node bin/clean-useless-files",
+    "zip": "node bin/zip",
+    "release": "npm run clean && npm run readme && npm run build && npm run clean-files && npm run makepot && npm run version && npm run zip"
   },
   "dependencies": {
-    "@wordpress/hooks": "^2.12.3",
+    "@wordpress/hooks": "^3.34.0",
     "chartjs-adapter-moment": "^1.0.1",
-    "debounce": "^1.2.1",
     "papaparse": "^5.4.1",
-    "vue": "^2.7.14",
-    "vue-chartjs": "^3.5.1",
-    "vue-content-loading": "^1.6.0",
-    "vue-multiselect": "^2.1.7",
-    "vue-notification": "^1.3.20",
-    "vue-wp-list-table": "^1.3.0",
-    "vue2-daterange-picker": "^0.6.8"
+    "vue-notification": "^1.3.20"
```

Comment:
this package is loaded as vue plugin level, so we can move this into devDevendencies